### PR TITLE
Updating SKE page

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/4e5vB6HS/3889-tweak-subject-knowledge-enhancement-ske-page-following-policy-feedback

### Context

After reviewing our new eligibility page, we got some feedback from policy (Gaby) that:

- candidates need to complete their SKE before ITT in order to receive both a SKE and ITT bursary
- rather than saying 'if you haven't used your subject knowledge in a while', it's more accurate to say 'if you completed your degree more than 5 years ago'

### CODE REVIEW

**Comment by gemmadallmandfe** [comment]
Looks good to me - happy to approve/merge once policy have given the go-ahead

**Comment by EllieNodder** [comment]
@gemmadallmandfe have made another tweak after feedback from Gaby - she's now happy for this to be merged. Other team members from across SKE have also provided more feedback on the overall page, but we've agreed to tackle this in another ticket as it seems like a wider piece of cross-team work.

**Comment by EllieNodder** [comment]
@gemmadallmandfe Gaby's now asked us to hold off merging this

**Comment by gemmadallmandfe** [comment]
Gaby still awaiting approval from SKE policy team - we've been asked to put this on hold until January

File: app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md
Comment (EllieNodder) [inline app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md:82]: @gazcarr @ethax-ross I've added the inset text for the international content (as per the ticket [#4053](https://trello.com/c/IQqvPIp5/4053-consider-design-solution-to-highlight-differences-for-international-teachers-and-trainees)) but it's coming up as the yellow highlighted indented text. Not sure how to change it to match your design Gary so it's indented but without the yellow background?

File: app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md:29]: do we have to say `(one of the following)` here @EllieNodder ?

File: app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md:63]:
```suggestion
They're usually online, but some can be classroom-based and some can be a mixture of both. 
```
does this need a hyphen @EllieNodder ?

Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md:63]:
```suggestion
SKE courses are usually online, but some can be classroom based and some can be a mixture of both. 
```

File: app/views/content/train-to-be-a-teacher/promos/_find-your-ske-course.html.erb
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_find-your-ske-course.html.erb:11]: should we say `subject knowledge enhancement` in full in either the heading or body @EllieNodder ?

File: app/views/content/train-to-be-a-teacher/promos/_find-your-ske-course.html.erb
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_find-your-ske-course.html.erb:15]:
```suggestion
      <a class="button" href="https://www.gov.uk/government/publications/subject-knowledge-enhancement-course-directory/subject-knowledge-enhancement-ske-course-directory">Search for a SKE course</a>
```
should we add `SKE` to the link text, to make it different from other Find buttons @EllieNodder ? might be useful for GA tracking to distinguish the links from those on other pages?

File: app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md:61]: We have some headings that are questions and some which aren't - should we make them consistent @EllieNodder (either one way or the other - whichever makes more sense!)

File: app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md:59]: @EllieNodder The video feels _slightly_ weird here (and the primary maths sentence feels a bit lost as a result) - does it need some kind of intro line before the video (that might not be good practice though!)?

**Review by gemmadallmandfe — COMMENTED** [review]
This looks great @EllieNodder - much improved!

I've left a couple of very minor comments but happy for you to ignore as needed

File: app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md
Comment (philspray) [reply app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md:59]: I agree, I think the video needs an intro line.

File: app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md
Comment (EllieNodder) [reply app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md:63]: I don't _think_ so? I would say that it's only hyphenated if followed by a noun - for example 'classroom-based course'. Paging @philspray for a grammar and punctuation opinion?

Comment (EllieNodder) [reply app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md:63]: I just changed this here because I felt like I was saying 'SKE courses' too many times (also not very SEO friendly to have the same phrase repeated so much)

Comment (philspray) [reply app/views/content/train-to-be-a-teacher/subject-knowledge-enhancement.md:67]: I would agree @EllieNodder, but I wonder if we need to say 'some courses can be classroom based'?

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me - happy for this to be merged when the policy team have approved

**Comment by EllieNodder** [comment]
Policy approval given by Gaby and SKE team

**Review by philspray — APPROVED** [review]
Looks good to me!

**Pull request merged**
